bdgenomics/workflows/mango_pipeline/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Licensed to Big Data Genomics (BDG) under one
```

Comment:
We're already in a repo called `workflows`, the `_pipeline` in the package name is redundant

Reply:
@heuermh do we want to rename all of them then?

Reply:
Eventually, I suppose.  This repo hasn't quite fully graduated to bigdatagenomics status yet, with full pull request, review, merge by someone other than yourself, etc. process.